system/Database/Postgre/Builder.php

```diff
@@ -351,15 +351,15 @@ protected function _truncate($table)
 	 *
 	 * @see https://www.postgresql.org/docs/9.2/static/functions-matching.html
 	 *
-	 * @param string|null $prefix
-	 * @param string      $column
-	 * @param string|null $not
-	 * @param string      $bind
-	 * @param boolean     $insensitiveSearch
+	 * @param string    $prefix
+	 * @param string    $column
+	 * @param string    $not
+	 * @param string    $bind
+	 * @param boolean   $insensitiveSearch
 	 *
 	 * @return string     $like_statement
 	 */
-	public function _like_statement(string $prefix = null, string $column, string $not = null, string $bind, bool $insensitiveSearch = false): string
```

Comment:
What's the reason for removing null values here? These have been this way for many, many years as part of CI3 and probably before. Unless you've confirmed that every use case for this method can work when those are not null, they need to stay. 

I get they don't follow best practices of having nullable arguments last, but if that's the only reason you're changing it, it needs to be fixed.

Reply:
This method is called only from only one place, from where all the params are getting passed to it.
So there is no need of declaring arguments as optional. Hence removed